BusinessLogic/Entities/EventDispatcher.cs

```diff
@@ -122,8 +122,11 @@ public void RegisterLocal(string eventName, Func<Event, HttpResponseMessage> cal
         /// of accepting a complete <see cref="Event"/>, it takes the event name and payload and proceeds to build
         /// an event out of these.
         ///
-        /// Optionally, also a dictionary of urlTemplateValues that will be used to replace the templateKeys in the
+        /// Optionally:
+        /// - urlTemplateValues dictionary that will be used to replace the templateKeys in the
         /// endpoint. By default this dictionary is null.
+        /// - subscriptions List<Subscription> that will be used to replace the local list of destinations used as
+        /// endpoint. By default the List is null.
```

Comment:
I like the description of these parameters, but is there a reason why we didn't add them inside a `<param>` tag as the parameters below? Eg: `<param name="eventName"></param>`